[TDX-5270] Snippets docs (#8672)
Kong/docs.konghq.com: app/dev-portal/portals/customization/custom-pages.md

> *Coming soon: auto-generated navigation MDC components for your Page structure*

### Create a new Page

> * Click to create a new Page at the top of the left sidebar.

Reviewer: ```suggestion
* In the Dev Portal editor, click **New page** in the sidebar.
```
